rfcs/models/identity/identity-update-policy.md

```diff
@@ -0,0 +1,69 @@
+
+A. The *Network DID* update request is attested by the participant who is leaving the network, that is the participant whose DID is being excluded from the  `networkParticipants` property. This indicates voluntary exit from the network.
+
+B. If condition A is not satisfied, and there exists an `updatePolicy` property of the *BlockchainNetworkMultiSig* `verificationMethod` in the *Network DID*, then the DID update request must be attested satisfying the `updatePolicy`.
+
+C. If condition A is not satisfied and there is no `updatePolicy` property of the *BlockchainNetworkMultiSig* `verificationMethod` in the *Network DID*, then the DID update request must be attested by strictly greater than half (`> N/3`) of the existing participants in the network indicated by `networkParticipants` property of the existing *Network DID* document.
+
+### Participants joining the network
+
+For new network participants joining the network, the DID update request must be authenticated based on the following conditions:
+
+A. If there is an `updatePolicy` property of the *BlockchainNetworkMultiSig* `verificationMethod` in the *Network DID*, then the DID update request must be attested satisfying the `updatePolicy`, and it must also be attested by the participant which is being added to the `networkParticipants`.
+
+B. If there is no `updatePolicy` property of the *BlockchainNetworkMultiSig* `verificationMethod` in the *Network DID*, then the DID update request must be attested by strictly greater than half (`> N/3`) of the existing participants in the network indicated by `networkParticipants` property of the existing *Network DID* document. The request must also be attested by the participant which is being added to the `networkParticipants` property.
+
+### Other updates to the Network DID
+
+For *Network DID* update requests, where the `networkParticipants` property is not altered, indicating that the participant list is unchanged, the requests are authenticated based on attestations satisfying `updatePolicy` property of the *BlockchainNetworkMultiSig* `verificationMethod`. If no `updatePolicy` is specified in the *Network DID*, then the update request has to be attested by strictly greater than half (`> N/3`) of the existing participants in the network indicated by `networkParticipants` property of the existing *Network DID* document.
+
+
+# Choice of `updatePolicy`
+
+The `updatePolicy` of a network is constructed by the network itself as a combination of different participants that need to attest any request to change the *Network DID* in future. Situations might arrive due to a poor choice of `updatePolicy` which cannot be satisfied by network the participants, where the *Network DID* can no longer be controlled by the network. In those cases the members of the network have to form a separate *Network DID* (a new network in the identity plane), and advertise the new DID as the new network address for the same network.
```

Comment:
"by the network itself as a combination of different participants" --> "through a consensus of different participants".
After this sentence, insert "The initial policy created for a Network DID creation requires unanimity, while subsequent policy updates are governed by existing policy.